flixel/system/frontEnds/CameraFrontEnd.hx

```diff
@@ -74,6 +74,34 @@ class CameraFrontEnd
 		cameraAdded.dispatch(NewCamera);
 		return NewCamera;
 	}
+	
+	/**
+	 * Inserts a new camera object to the game.
+	 *
+	 * @param	NewCamera         The camera you want to add.
+	 * @param	Position          The position in the list where you want to insert the camera
+	 * @param	DefaultDrawTarget Whether to add the camera to the list of default draw targets. If false, 
+	 *                            `FlxBasics` will not render to it unless you add it to their `cameras` list.
+	 * @return	This FlxCamera instance.
+	 */
+	public function insert<T:FlxCamera>(NewCamera:T, Position:Int, DefaultDrawTarget:Bool = true):T
```

Comment:
The official rule is to use lowercase arg names, we are slowly changing old uppercase args as we  go